### PR TITLE
cmd, core, eth: support for the olympic network

### PR DESCRIPTION
**Pull request by obscuren**
Added a `--olympic` flag which initialiser the olympic protocol settings

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Mon Aug 3 15:49:13 UTC 2015_

File: core/chain_manager.go [hunk 1]
```diff
@@ -528,6 +528,10 @@ func (self *ChainManager) InsertChain(chain types.Blocks) (int, error) {
 	self.chainmu.Lock()
 	defer self.chainmu.Unlock()
 
+	if len(chain) > 0 && chain[0].NumberU64() == 0 {
```
Comment (obscuren) [inline core/chain_manager.go]: TODO remove this

**Comment by Gustav-Simonsson** [comment]
:+1:

**Comment by pompomJuice** [comment]
:+1:

**Comment by bas-vk** [comment]
:+1:

**Pull request merged**